src/commands/xkcd.ts

```diff
@@ -0,0 +1,165 @@
+import {
+	ApplicationCommandOptionType,
+	EmbedBuilder,
+	InteractionReplyOptions,
+	MessageReplyOptions,
+} from 'discord.js';
+import axios from 'axios';
+
+// defining the response
```

Comment:
The definition on the next line makes clear that you're defining the response, lol. Why also say it in a comment?

Reply:
Guess it was an old mental sketch note that I left in (I do a lot of pseudocode comments that disappear by push time, this one must've slipped through the cracks.